lib/y2logsstep.pm

```diff
@@ -70,7 +70,7 @@ sub record_dependency_issues {
 }
 
 sub save_upload_y2logs() {
-    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2";
+    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2", 540;
```

Comment:
why is that necessary? This has an influence on all other tests that use this so we probably don't want to change it. But at least please tell us why this is necessary for you.